src/main/resources/helptexts/subjects.html

```diff
@@ -1,5 +1,27 @@
-<h1>Test header</h1>
-<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque semper nulla eu placerat consectetur.</p>
-<p>Sed dignissim dapibus venenatis. Sed rhoncus.</p>
-<h2>Subheader2</h2>
-<p>Maecenas luctus metus eleifend, pulvinar mi ac, pellentesque nibh. Sed feugiat, augue.</p>
+<h1>Subject (ABR Complex)</h1>
```

Comment:
This is the wrong helptext. If you don't have a specific one for `subjects`, just keep the h1, and remove the rest of the content